recipes/sphinx-reredirects/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2020, documatt
```

Comment:
Could you please open an issue in the upstream repository so that the LICENSE file is included in next version?

Not blocking. Recipe looks good. I'll merge.

Reply:
Done: https://gitlab.com/documatt/sphinx-reredirects/-/issues/4